cmd/repo_info.go

```diff
@@ -24,9 +24,10 @@ import (
 
 func init() {
 	repoInfoCmd := &cobra.Command{
-		Use:   "info",
-		Short: "Get info about a artifact",
-		Long:  `Get info about a artifact`,
+		Use:   "info [artifact...]",
+		Short: "Get info about an artifact",
+		Long:  `Get info about an artifact`,
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
I'm not sure I want to force a minium number of args...

Reply:
Does it make sense to run this command without any argument?